doc/how-to/replication/repl_leader_elect.rst

```diff
@@ -44,13 +44,11 @@ Configuration
 *   ``election_fencing_enabled`` -- switches the :ref:`leader fencing mode <repl_leader_elect_fencing>` on and off.
     For the details, refer to the :ref:`option description <cfg_replication-election_fencing_enabled>` in the configuration reference.
 
-Besides, it is important to know that
-being a leader is not the only requirement for a node to be writable.
-A leader node should have its :ref:`read_only <cfg_basic-read_only>` option set
-to ``false`` (``box.cfg{read_only = false}``),
-and its :ref:`connectivity quorum <cfg_replication-replication_connect_quorum>`
-should be satisfied (``box.cfg{replication_connect_quorum = <count>}``)
-or disabled (``box.cfg{replication_connect_quorum = 0}``).
+Note that a leader should also satisfy the following requirements:
```

Comment:
I'd probably clarify that these are the requirements for the leader to be writeable.

Otherwise it's not clear what 'a leader should satisfy the following requirements' means.

Reply:
The next line tells:

> The :ref:`read_only <cfg_basic-read_only>` option is to ``false``.

I thought that this means that the leader is writeable. Or do you mean smth else?

Reply:
> I thought that this means that the leader is writeable.
 
 `box.cfg.read_only == false` is only one of the many prerequisites for the leader to be writeable (another prerequisite is not being an orphan, for example).

> Or do you mean smth else?

I think the old wording described this better (that there are multiple reasons to be read-only):
`being a leader is not the only requirement for a node to be writable`

Reply:
Thanks! Hope, now I got it right :)
![image](https://user-images.githubusercontent.com/38073144/230936757-5b722b1c-b500-4a58-981b-002f77fc71db.png)

Reply:
Now it's fine, thanks!